src/test/java/net/snowflake/client/jdbc/CustomProxyIT.java

```diff
@@ -1,40 +1,45 @@
 package net.snowflake.client.jdbc;
 
 import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertTrue;
 import static junit.framework.TestCase.fail;
 import static net.snowflake.client.AbstractDriverIT.getFullPathFileInResource;
+import static net.snowflake.client.jdbc.SnowflakeDriverIT.findFile;
 import static net.snowflake.client.jdbc.SnowflakeUtil.systemGetProperty;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
 
+import java.io.File;
 import java.net.Authenticator;
 import java.net.PasswordAuthentication;
-import java.net.URL;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.Properties;
 import net.snowflake.client.category.TestCategoryOthers;
 import net.snowflake.common.core.SqlState;
 import org.junit.Ignore;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.junit.rules.TemporaryFolder;
+
+// To run these tests, you must:
+// 1.) Start up a proxy connection. The simplest ways are via Squid or BurpSuite
+// 2.) Enter your own username and password for the account you're connecting to
+// 3.) Adjust parameters like role, database, schema, etc to match with account accordingly
```

Comment:
I am not fully understand these comments. From the comments, it looks like if we don't do the setup, then we are not able to run following tests. But to my understanding, those tests are keep running in our CI? Do we do some setup in our CI env for them?

Reply:
These tests are run manually only. We ignore all of them for both Jenkins and GithubActions because we don't have a way to set up a proxy connection in those testing environments. So I just added some comments to try to help users who want to manually run these tests (since I forgot the process from the last time I tried). But I could add another clarifying comment that these are manual tests!

Reply:
Oh my bad, I didn't see the `@Ignore`. Sounds good!